### PR TITLE
Added support for exporting PFM files.

### PR DESCRIPTION
**Pull request by StanleyMines**
Export to PFM directly from Chunky, without needing to complete a render with the Denoise plugin.

PfmFileWriter class code is modified from PFM saving code in @leMaik's Denoise Plugin.

### CODE REVIEW

File: chunky/src/java/se/llbit/pfm/PfmFileWriter.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        out.write(ByteBuffer.allocate(4)
+            .order(byteOrder)
+            .putFloat((float)pixel[1])
+            .array());
+        out.write(ByteBuffer.allocate(4)
+            .order(byteOrder)
+            .putFloat((float)pixel[2])
+            .array());
+      }
+    }
```
Comment (aTom3333) [inline chunky/src/java/se/llbit/pfm/PfmFileWriter.java:86]: This feels inefficient :/ (i don't know for sure without measuring), i'd like to experiment a bit with it

**Comment by aTom3333** [comment]
Seems correct to me

File: chunky/src/java/se/llbit/pfm/PfmFileWriter.java (hunk 1 above)
Comment (StanleyMines) [reply chunky/src/java/se/llbit/pfm/PfmFileWriter.java:86]: It does feel a bit inefficient, but it is how leMaik's Denoiser plugin implemented it.

**Comment by aTom3333** [comment]
So I had a quick look about the inefficient writing, it can easily be improved by writing to the file row by row.
I join the changes as a patch (i had to rename it to .txt because of github) if you want to apply it before your PR is merged (otherwise, i'd make a PR later)
[Write_to_pfm_row_by_row.txt](https://github.com/chunky-dev/chunky/files/5339978/Write_to_pfm_row_by_row.txt)
That's a simple change that makes the writing almost 10 times faster (from my single data point, a 1080*1920 render), not that it matters much in the grand scheme of things but that's always nice

**Comment by leMaik** [comment]
@StanleyMines Thanks for the PR, looks good! :tada: 

@aTom3333 Since you are a contributor to this repository, you should be able to commit directly to @StanleyMines' branch. If the scene has post-processing disabled, the call to `postProcessRow` could be skipped entirely.

**Comment by StanleyMines** [comment]
Suggestions from @aTom3333 and @leMaik implemented (not leMaik's 2nd one though. Is quite a bit faster than before though (far faster than TIFF)).

**Review by leMaik — APPROVED** [review]

**Review by leMaik — APPROVED** [review]

**Comment by leMaik** [comment]
@StanleyMines Nice first PR! :rocket:

**Pull request merged**